### ISSUE TITLE
BadZipFile issue

### ISSUE THREAD

**Issue by barthchristian** [admin-shell-io/aas-test-engines#10]
I'm trying to run the aas_test_engine as a minimal example to test an aasx file (metamodel v3) previously created and stored with the package explorer. I get the "File is not a zip file" error, despite the file being a zip file (but not called .zip) :).  The setup is a windows computer, running anaconda3 as python environment. Anyone has an idea how to fix this?
![image](https://github.com/admin-shell-io/aas-test-engines/assets/48819812/11ddc72d-4ef7-4330-9267-643b5f2114b8)

**Comment by otto-ifak** [comment]
Thanks for reporting this. Actually, the README is wrong, here, it should be:

```python
with open('aas.aasx', 'rb') as f:
    result = file.check_aasx_file(f)
# result.ok() == True
```

(Note the extra `'rb'`)

The command line tool did this wrong, too, has been fixed with aa8f0489e8fc83c3af608d9bf0102b310a9e3a2d.

Does this resolve your issue?

**Comment by barthchristian** [comment]
Hi thanks for the reply, this indeed fixes my issue!
